### ISSUE TITLE
"Unable to start terminal process: CreateProcess failed" when trying to open terminal on Windows

### ISSUE THREAD

**Issue by DanTup** [microsoft/vscode#75885]
I'm on a fairly clean install of Windows 10 Pro. When I chit `Ctrl`+`'` to bring up the terminal, it just shows an empty terminal window that never shows a prompt (and doesn't accept input):

![terminal](https://user-images.githubusercontent.com/1078012/59916224-7d117c00-9416-11e9-8033-be27a1585a65.png)

In the dev tools console I see this:

```
 Uncaught Error: Unable to start terminal process: CreateProcess failed
    at new WindowsPtyAgent (c:\Users\danny\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\node-pty\lib\windowsPtyAgent.js:63:36)
    at new WindowsTerminal (c:\Users\danny\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\node-pty\lib\windowsTerminal.js:49:24)
    at Object.spawn (c:\Users\danny\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar\node-pty\lib\index.js:28:12)
    at g.setupPtyProcess (file:///C:/Users/danny/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:2699:174)
    at new g (file:///C:/Users/danny/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:2698:824)
    at c._createInstance (file:///C:/Users/danny/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:1390:671)
    at createInstance (file:///C:/Users/danny/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:1390:92)
    at Proxy.createTerminalProcess (file:///C:/Users/danny/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:3880:183)
    at C._launchProcess (file:///C:/Users/danny/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:3865:754)
    at C.createProcess (file:///C:/Users/danny/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:3864:318)
    at setTimeout (file:///C:/Users/danny/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:3705:196)
```

I have no settings relating to the terminal in my VS Code settings and expect it to use PowerShell by default. I can launch PowerShell without any issues from the task bar and start menu. I do have a PoSh profile that defines some functions, but I tried removing them all and it made no difference.

I'm not sure how to debug any further.

**Comment by DanTup** [comment]
Oh whoops, I found out what it is. It's sort-of user error, but I'll leave this open in case you want to improve the way it fails.

The folder I had open in VS Code had a typo in its name, so I renamed it (in Windows explorer) and forgot to re-open it. So VS Code's "open folder" actually didn't exist. Presumably the terminal was failing to launch because you were trying to spawn it in a non-existant folder!
